### ISSUE TITLE
The graph should show the median task duration

### ISSUE THREAD

**Issue by nivdror** [apache/airflow#39937]
### Description

Hi,
I would like when hovering over a task in the graph UI to be shown the median duration of that task.
I seen that from airflow 2.9 you got that data displayed on a different tab.

### Use case/motivation

I'm trying to offer my data-engineers a better visibility over their flows. to be able to extract ETA of those flows.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by bbovenzi** [comment]
We show this in the Task Duration tab. And we just expanded the taks duration tab to see across all runs/tasks. 

What's the use case here? To see how far off the task duration is from the median? That could be interesting to show.

**Comment by nivdror** [comment]
Hi @bbovenzi ,
Thanks for replying.

The use case here is simple. Think of a case when your ETL is delaying for some reason and you got SLA alert.
You will need provide an estimate of the tasks that haven't run yet until you are ready to publish your data.

This feature will give the DE oncall with that capability without running into extra hassle (with viewing the previous runs).

I hope I was more clear now

**Comment by bbovenzi** [comment]
Yes that makes sense. I was thinking that, when no run is selected the Gantt chart could render a summary. It would look like a box+whisker chart to show the median, and min/max durations. We could also then use that data to show how a single task instance or dag run stacks up.

**Comment by vatsrahul1001** [comment]
@bbovenzi @pierrejeambrun Are we planning this for 3.3.0 if not can be moved to 3.4.0
